ztp/siteconfig-generator/siteConfig/siteConfigHelper.go

```diff
@@ -0,0 +1,129 @@
+package siteConfig
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	machineconfigv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
+	mcfgctrlcommon "github.com/openshift/machine-config-operator/pkg/controller/common"
+
+	"github.com/lack/yamltrim"
+	yaml "gopkg.in/yaml.v3"
+)
+
+const mcRoleKey = "machineconfiguration.openshift.io/role"
+const mcName = "predefined-extra-manifests"
+
+func isExcluded(s []string, str string) bool {
```

Comment:
This is a useful function, but maybe better named `listContains`, since it really doesn't have anything to do with exclusion per se?

Also (optional), if you use a `map[string]bool` instead of a `[]string`, you don't need this function at all and the code below could just be:
```
if excludes[k] {
  continue
}
```

(Assuming you insert them as `excludes["filename"] = true` when you populate the map)

This is a cool feature of go maps -> The default value of `bool` is false, and if you only call the map get function with a single-argument return, you get the default value if the key doesn't exist in the map!

Example: https://go.dev/play/p/vZVaxZxXyjm

Reply:
Thanks for the suggestion.